packages/cli/lib/init-global.ts

```diff
@@ -11,14 +12,26 @@ export function initGlobal() {
 
   if (workspace) {
     // Found a workspace root - hand off to the local copy of Nx
-    require(path.join(
+    const localNx = path.join(
       workspace.dir,
       'node_modules',
       '@nrwl',
       'cli',
       'bin',
       'nx.js'
-    ));
+    );
+    if (fs.existsSync(localNx)) {
+      require(localNx);
+    } else {
+      output.log({
+        title: `The current directory isn't part of an Nx workspace.`,
+        bodyLines: [
+          `To convert an Angular workspace to Nx run:`,
+          chalk.bold.white(`ng add @nrwl/workspace`),
+        ],
+      });
+      process.exit(0);
```

Comment:
I think this should be non-zero since it didn't actually do the thing the user wanted to do :thinking:?

Reply:
Yup, fair enough. Might change this to an error message then.